.pre-commit-config.yaml

```diff
@@ -15,6 +15,11 @@ repos:
       - id: check-yaml
       - id: check-toml
 
+  - repo: https://github.com/astral-sh/uv-pre-commit
```

Comment:
This is nice, can you add it https://github.com/VectorInstitute/aieng-template-uv/blob/main/.pre-commit-config.yaml as well?